### ISSUE TITLE
pre_trained model "resnext-101-kinetics-hmdb51_split1.pth"

### ISSUE THREAD

**Issue by hareeshdevarakonda** [kenshohara/3D-ResNets-PyTorch#66]
Is this pre_trained model on hmdb51 dataset? could you please check it, because resnext-101-kinetics-hmdb51_split1.pth is working on --n_finetune_classes 101 only but not on 51 is it true?
Could you please check and let me know because i did not got accuracies what you reported in paper on this pre_trained model

**Comment by kenshohara** [comment]
I confirmed the file but the model file worked when I set only --n_classes 51.
How do you execute?

**Comment by hareeshdevarakonda** [comment]
I am asking finetune classes actually classes 51 are fine
Command I used
sudo python3 main.py  --root_path 3D-ResNets-PyTorch-master --video_path hmdb51_jpeg --annotation_path /hmdb51_1.json --result_path result_hmdb_100 --dataset hmdb51 --model resnext --model_depth 101 --n_classes 51 **_--n_finetune_classes 101_** --batch_size 50 --n_threads 4 --pretrain_path resnext-101-kinetics-hmdb51_split1.pth --resnet_shortcut B --resnext_cardinality 32

I am not getting the accuracy what you reported in paper
